_config.yml

```diff
@@ -95,6 +95,7 @@ JB :
       colorscheme: light
    
   # Settings for analytics helper
+  # Set 'site.safe' to true to enable analytics plugin
```

Comment:
Hi ground0g, I've already deleted my fork repo, cannot make a new pr now.. It would be good if you can fix it with your account.